apps/www/app/glossary/client.tsx

```
@@ -0,0 +1,232 @@
+          {/* Left Sidebar */}
+          <div>
+            <h2 className="w-full mb-4 font-semibold text-left blog-heading-gradient">
+              Find a term
+            </h2>
+            <SearchInput
+              placeholder="Search"
+              className="rounded-lg mb-4 border-[.75px] border-white/20 lg:w-[232px]"
+            />
+            <Accordion type="multiple" className="space-y-2">
+              {categories.map((category) => (
+                <Fragment>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)